### ISSUE TITLE
Excessive Memory usage after upgrading to DebugKit 4.8.0 (from 4.7.1) in Testsuite

### ISSUE THREAD

**Issue by swiffer** [cakephp/debug_kit#871]
Memory usage / Execution Time in DebugKit is crazy after updating to DebugKit 4.8.0.
Anyone else noticing this issue? It's a test for one controller only - running the full TestSuite results in out of memory errors

with DebugKit 4.7.1
```bash
> composer test -- --filter PagesControllerTest
> phpunit --colors=always --filter PagesControllerTest
PHPUnit 9.5.20 #StandWithUkraine

......                                                              6 / 6 (100%)

Time: 00:00.395, Memory: 18.00 MB

OK (6 tests, 15 assertions)
```

with DebugKit 4.8.0
```bash
> composer test -- --filter PagesControllerTest
> phpunit --colors=always --filter PagesControllerTest
PHPUnit 9.5.20 #StandWithUkraine

......                                                              6 / 6 (100%)

Time: 00:01.587, Memory: 414.00 MB

OK (6 tests, 15 assertions)
```

**Comment by LordSimal** [comment]
Sorry, this is now fixed in the new 4.8.1 version. 
Please update your Debug Kit and this problem should be gone.

**Comment by swiffer** [comment]
Thanks addressing this issue so quickly!
